docs/detections/how-to-query-alert-indices.asciidoc

```diff
@@ -0,0 +1,60 @@
+[[how-to-query-alert-indices]]
+[role="xpack"]
+== How to query alert indices
```

Comment:
What about something like "Alert indices and aliases" (or "Detection alert indices and aliases") ? It might not be necessary to identify in the title that this is a "best practices" sort of document. Internal headers and or introductory paragraphs can convey that.

Reply:
Let's go with "Query alert indices".